schema/docs/versions.md

```diff
@@ -288,6 +288,78 @@ Now that we know how to encode version objects, that would be written as:
 		}
 	]
 
+### Version Types
+
+#### Semantic versioning 2.0.0
+
+Type identifier: `semver-2.0.0`
+Formally specified here at https://semver.org/spec/v2.0.0.html
+`semver-2.0.0` is new type introduced to formally specify usage of semantic versioning. 
```

Comment:
Nit: "new type" &rarr; "type"

Reply:
Good call. Removed in https://github.com/CVEProject/cve-schema/pull/371/commits/c9fde5054e8815efe8294aab9a896e5f7004cb22